charts/index.yaml

```diff
@@ -3,20 +3,29 @@ entries:
   csi-driver-nfs:
```

Comment:
Do we need to update https://github.com/kubernetes-csi/csi-driver-nfs/tree/master/deploy? Seems like having 2 different copies of the spec will lead to confusion.